test/unit/spec/appWidgets/input/select2ObjectinputSpec.js

```diff
@@ -189,12 +189,16 @@ define([
 
         it('Should start and stop', (done) => {
             const widget = Select2ObjectInput.make(testConfig);
+            spyOn(widget, 'start').and.callThrough();
+            spyOn(widget, 'stop').and.callThrough();
             widget
                 .start({ node: node })
                 .then(() => {
+                    expect(widget.start).toHaveBeenCalled();
```

Comment:
That's a little redundant, I think. I mean, you're calling start. So, yes, start was called. And it returned a Promise, which is why that hunk of code was executed. Is there anything else you can trace on? Something in the node you give it?

Reply:
I was being rather lazy there! I'll find a better test.